config/manager/manager.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         app.kubernetes.io/component: kusk-gateway-manager
     spec:
       securityContext:
-        runAsNonRoot: true
+        runAsNonRoot: false
```

Comment:
@jasmingacic, this is far from ideal.

What's the `kustomize` way of adjusting this value?